packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -382,11 +422,35 @@ const GlobalHeader = ({ className, activeSite }) => {
               />
             </li>
             {locales.length > 1 && (
-              <li>
+              <li
+                css={css`
+                  @media screen and (max-width: 545px) {
```

Comment:
I noticed this `545px` value is used in a few places. It would be nice to pop this out to a variable which would make it easier to change later and remove the "magic number"-ness by giving it a name.